### ISSUE TITLE
Start Script

### ISSUE THREAD

**Issue by zacharylove** [Daddy-s-Dungeons-Tools/ddtools-web#25]
It would be nice to have a script that runs both the backend and frontend servers in two separate terminal instances. This would save the effort of opening two terminals and running two commands- perfect for lazy developers like us :)

My plan is to add a launch script in `.vscode/tasks.json` to automate this process. Unfortunately, this approach will only work for Visual Studio Code, so if anyone finds a better alternative, please let me know in the comments.

@Apexal, what are the commands we need to call to properly run both servers?

**Comment by Apexal** [comment]
In the `frontend/` folder `npm start`
In the `functions/` folder `npm start`

**Comment by Apexal** [comment]
Yep! The `functions/` npm start starts the Firebase emulators

**Comment by zacharylove** [comment]
Made an [initial commit](https://github.com/Daddy-s-Dungeons-Tools/ddtools-web/compare/master...start-scripts) using the [concurrently](https://www.npmjs.com/package/concurrently) npm package to run both servers simultaneously in the same terminal. 
I added some custom names and colors to pretty it up a bit. Just needs some testing and we're good for a PR.
Note that this is an additional dependency, so if this branch gets merged into `master`, everyone will need to `npm install` it.

The only issue I see right now is #26, but just to be safe, can someone test the start script on a non-mac device?

**Comment by Apexal** [comment]
> Also, oddly enough, running Firebase functions requires Java to be installed and added to the system's PATH. We may want to add that to our documentation just in case, as the project will not run without it.

Good catch. Feel free to add it to the README under https://github.com/Daddy-s-Dungeons-Tools/ddtools-web#local-setup

**Comment by zacharylove** [comment]
I added some notes about Java and Firebase to the ["Getting Started" portion of the wiki](https://github.com/Daddy-s-Dungeons-Tools/ddtools-web/wiki/Getting-Started/f47214cd265fa191848b6a858f78fd0b77c550c8) for now, since I'd rather not create another branch/PR for such a minor change. 

> I'd recommend adding a new npm script called "start-both" or similar instead of using "start", as there are use cases where we just want to run the React app alone with "start".

And yeah, I agree with that. I'll make a PR for the start scripts soon, just after I verify that they didn't break anything.
